docs/eventing/sink/README.md

```diff
@@ -0,0 +1,46 @@
+A Sink is an Addressable resource that acts as a link
+between the Eventing mesh and an entity or system.
+
+We can connect any `Source` to a `Sink`, like `PingSource` and `KafkaSink`:
+
+```yaml
+apiVersion: sources.knative.dev/v1beta1
+kind: PingSource
+metadata:
+  name: test-ping-source
+spec:
+  schedule: "*/1 * * * *"
+  jsonData: '{"message": "Hello world!"}'
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1alpha1
+      kind: KafkaSink
+      name: my-kafka-sink
+```
+
+We can connect a `Trigger` to the `Sink`, so that we can filter events, before sending them to a `Sink`:
```

Comment:
`Trigger`  -> "`Trigger` object"
`Sink` -> "sink"
please do this for API objects and concepts throughout as mentioned in prev comment